sparkle/src/components/CardButton.tsx

```diff
@@ -69,15 +55,21 @@ export function CardButton({
   rel = "",
   replace,
   shallow,
+  ...props
 }: CardButtonProps) {
   const { components } = React.useContext(SparkleContext);
   const Link: SparkleContextLinkType = href ? components.link : noHrefLink;
 
+  const cardButtonClassNames = cn(
+    cardButtonVariants({ variant, size }),
+    className
+  );
+
   if (href) {
     return (
       <Link
         href={href}
-        className={cardButtonVariants({ variant, size, className })}
+        className={cardButtonClassNames}
         replace={replace}
         shallow={shallow}
         target={target}
```

Comment:
We should now be able to actually remove the Link and pass all those props to the button

Reply:
Wondering why we are not doing that ⬆️

Reply:
I actually tried and our Button doesn't render "children" properly (we actually should not let you do that btw)

Reply:
not super satisfying but I'll take it!

Reply:
@aubin-tchoi Well we don't really want to be able to pass any kinda react node inside a Button right?

Reply:
Open to alternatives though 🤓

Reply:
yeah sure, but I don't like that we would want to pass children here, I feel like there is something wrong with this component being called a Button

Reply:
Agree on the naming, I would rather call it ClickableCard which is all it does in the end, right? 😅

Reply:
Which I suspect, is the reason we use to have a div and not button. We could also rollback to it?

Reply:
Leaning toward a rollback yeah